apps/_scaffold/templates/layout.html

```diff
@@ -58,6 +58,6 @@
   <script src="js/axios.min.js"></script>
   <script src="js/vue.min.js"></script>
   <script src="js/utils.js"></script>
-  [[if globals().get('flash'):]]<script>utils.flash([[=XML(flash)]]);</script>[[pass]]
+  [[if globals().get('flash'):]]<script>utils.flash([[=XML("\"%s\"" % flash) if isinstance(flash, str) else XML(flash)]]);</script>[[pass]]
```

Comment:
Not fine with this. It was cryptic enough as it was :-(

Reply:
Yes, it looks not perfect, but `utils.flash([[=XML("hello world")]])` turns into `utils.flash(hello world)`, that causes an js-error